scylla-cql/src/frame/value_tests.rs

```diff
@@ -110,13 +109,14 @@ fn cql_varint_normalization() {
     }
 }
 
+#[cfg(feature = "num-bigint-03")]
 #[test]
-fn cql_varint_normalization_with_bigint() {
+fn cql_varint_normalization_with_bigint03() {
```

Comment:
Nit: there is no corresponding test for `0.4`. I just checked that the normalization behavior is the same in `0.4`, fortunately. The test case can be added in a follow-up.

Reply:
Yes, I've also checked that normalization works the same in this case. Wasn't sure if such test case is redundant or not.